core/scss/elements/table/_table.scss

```diff
@@ -14,9 +14,37 @@
 // Style guide: Elements.Table
 //
 table {
-  @include tables;
```

Comment:
I could swing both ways on this one. Perhaps a mixin is useful? Or just leave it up to the deault styling to do the job. Thoughts?

Reply:
While I lean towards leaving it up to the default styling (and adding additional modifiers as needed) for tags where we can, I can also swing either way here. If we feel that something is reusable, I am game for keeping it in a mixin. Again, happy to go either route.

Reply:
Ok, let's leave it like this for now. We can refactor later if we need to.